hack/make/.build-deb-ce/control

```diff
@@ -0,0 +1,12 @@
+Source: docker-ce
+Section: admin
+Priority: optional
+Maintainer: Docker <support@docker.com>
+Standards-Version: 3.9.6
+Homepage: https://dockerproject.org
```

Comment:
good question! i have no opinion -- i just copied it from the other control file :)

if we get another vote for this i'm happy to make the adjustment.